intellij/src/saros/intellij/negotiation/ModuleUtils.java

```diff
@@ -28,10 +27,18 @@
   @Nullable
   static Path getRelativeRootPath(@NotNull Path basePath, @NotNull SourceFolder sourceFolder) {
     String sourcePath = sourceFolder.getUrl();
+
+    // Workaround to convert the Intellij "URI" to a real URI
```

Comment:
Is this still a workaround? Even if it is debatable practice to work with an own URI definition in intellij, we are using the APIs as intended and as far as I understand the APIs are working as expected. My perception is that this not a workaround anymore.
(Might be a bit pedantic and irrelevant. Therefore, approve :+1:)

Reply:
I would still like to mention the fact that their "URL" is not actually a standard URI as this might be an important detail for someone having a look at the logic later on. I don't think the word "workaround" makes a big difference here, but I can remove it if you prefer.

Reply:
You don't have to remove it.